[IMP] social_media, *: add discord to the social networks

shsa-odoo opened the pull request:
This PR implements the following improvements:

1. Add Discord to Social Networks:
This update adds Discord as a new social network within the application. Users can now link their Discord profiles to their websites. The templates and social media section have been modified to seamlessly integrate Discord alongside existing networks.

2. Replace GitHub with Facebook in Website Templates:
Replaced GitHub links with Facebook links in the social media section of the website templates to better align with our current marketing strategy. Updated the GitHub icon and link to Facebook in both "Descriptive" and "Minimalist" templates. This change aims to boost engagement on platforms where our target audience is more active.

[*]: web, website, website_blog, website_event.

task-3503084

robodoo commented:
[![Pull request status dashboard](https://mergebot.odoo.com/odoo/odoo/pull/174528.png)](https://mergebot.odoo.com/odoo/odoo/pull/174528)

msh-odoo commented on addons/website/static/tests/tours/snippet_social_media.js:
```diff
@@ -89,9 +89,9 @@ registerWebsitePreviewTour('snippet_social_media', {
-    ...addNewSocialNetwork(7, 7, 'https://www.youtu.be/y7TlnAv6cto'),
+    ...wTourUtils.dragNDrop({id: 's_social_media', name: 'Social Media'}),
+    ...wTourUtils.clickOnSnippet({id: 's_social_media', name: 'Social Media'}),
+    ...addNewSocialNetwork(8, 8, 'https://www.youtu.be/y7TlnAv6cto'),
```
Maybe we should dragAndDrop discord icon lilke we add other icons.

shsa-odoo replied:
Hello @msh-odoo i am not sure what is expected here, but on dragAndDroping the `s_social_media` will definitely have the discord icon there hence the count adds up to 8. Or is there any other context are you talking about ?

bvr-odoo commented:
@shsa-odoo could you rebase please

bso-odoo commented:
@robodoo delegate=bvr-odoo

shsa-odoo commented:
@bvr-odoo updated it.

shsa-odoo commented:
@bvr-odoo This is happening because we have updated [s_social_media](addons/website/views/snippets/s_social_media.xml) as we have added discord to it and updated data-vxml version.

bvr-odoo commented:
> @bvr-odoo This is happening because we have updated [s_social_media](https://github.com/odoo/odoo/pull/174528/files#diff-2e2f9f480e46cd1ec57665c16b15679cde5c5d98419eed80f5abd1adfe63ad91R9) as we have added discord to it and updated data-vxml version.

Yes, but why do we need that VXML? It wasn't done when the TikTok icon was added.
Could you check if it's really necessary

shsa-odoo commented:
> > @bvr-odoo This is happening because we have updated [s_social_media](https://github.com/odoo/odoo/pull/174528/files#diff-2e2f9f480e46cd1ec57665c16b15679cde5c5d98419eed80f5abd1adfe63ad91R9) as we have added discord to it and updated data-vxml version.
> 
> Yes, but why do we need that VXML? It wasn't done when the TikTok icon was added. Could you check if it's really necessary

Here's a refined and professional version of your response:  

---
@bvr-odoo 
In my opinion, we should have added the `data-vxml` update when TikTok was introduced. Did we intentionally skip adding it at that time?  

@Guillaume-gdi, could you please help us here ? I went through the conversation in the related PR ([[#116837](https://github.com/odoo/odoo/pull/116837)](https://github.com/odoo/odoo/pull/116837)) but didn’t find any reference to it.  

Regarding the current update, adding the `data-vxml` version is essential to ensure that users who had previously added this snippet and later migrated to the latest version are properly notified that their snippet is outdated. Without this update, they wouldn’t receive such a notification, potentially causing issues with outdated content.

bvr-odoo commented:
> > > @bvr-odoo This is happening because we have updated [s_social_media](https://github.com/odoo/odoo/pull/174528/files#diff-2e2f9f480e46cd1ec57665c16b15679cde5c5d98419eed80f5abd1adfe63ad91R9) as we have added discord to it and updated data-vxml version.
> > 
> > 
> > Yes, but why do we need that VXML? It wasn't done when the TikTok icon was added. Could you check if it's really necessary
> 
> Here's a refined and professional version of your response:
> 
> @bvr-odoo In my opinion, we should have added the `data-vxml` update when TikTok was introduced. Did we intentionally skip adding it at that time?
> 
> @Guillaume-gdi, could you please help us here ? I went through the conversation in the related PR ([[#116837](https://github.com/odoo/odoo/pull/116837)](#116837)) but didn’t find any reference to it.
> 
> Regarding the current update, adding the `data-vxml` version is essential to ensure that users who had previously added this snippet and later migrated to the latest version are properly notified that their snippet is outdated. Without this update, they wouldn’t receive such a notification, potentially causing issues with outdated content.

@shsa-odoo 
GDI is no longer part of the website team ;)
> potentially causing issues with outdated content ?
Did you test it ? if there are no issue, vxml is not necessary.
Anyway, we have to avoid it and handle this in JS or another way in needed, but this message should be avoided in most cases.

shsa-odoo commented:
@bvr-odoo I have updated the changes. vxml for this case was actually never needed.

bvr-odoo commented:
@odoo/rd-framework-js 
`Check Style` is failing on the runbot due to the use of an `@extend` ([here](https://github.com/odoo/odoo/pull/174528/files#diff-734d96a0e99f6c21e7e3d5a4541da4f580f3fc57df5e45c71c055c963e15cd9bR81))
The same rule is already used multiple times in the same file without any issues.
Could we get approval to bypass the check?

adr-odoo commented:
@robodoo override=ci/style

robodoo commented:
@adr-odoo you are not allowed to override 'ci/style'.

adr-odoo commented:
@robodoo override=ci/style

bvr-odoo commented:
@robodoo rebase-ff r+

robodoo commented:
Merge method set to rebase and fast-forward.
